AudioKit/Common/Internals/AudioKit.swift

```diff
@@ -205,79 +212,82 @@ public typealias AKCallback = () -> Void
     // MARK: - Start/Stop
 
     /// Start up the audio engine with periodic functions
-    open static func start(withPeriodicFunctions functions: AKPeriodicFunction...) {
+    open static func start(withPeriodicFunctions functions: AKPeriodicFunction...) throws {
         for function in functions {
             function.connect(to: finalMixer)
         }
-        start()
+        try start()
     }
 
     /// Start up the audio engine
-    @objc open static func start() {
+    @objc open static func start() throws {
         if output == nil {
-            AKLog("AudioKit: No output node has been set yet, no processing will happen.")
+            AKLog("No output node has been set yet, no processing will happen.")
```

Comment:
This should throw a custom error.  If a function can throw, than it should when failing.